dev_guide/managing_images.adoc

```diff
@@ -0,0 +1,582 @@
+----
+====
+
+Querying external registries to synchronize tag and image metadata is not
+currently an automated process. To resynchronize manually, run:
+
+----
+$ oc import-image <image_stream_name>
+----
+
+Within a short amount of time, OpenShift will communicate with the external
+registry to get up to date information about the Docker image repository
+associated with the image stream.
+
+[[insecure-registries]]
+=== Importing Images from Insecure Registries
+
+An image stream can be configured to import tag and image metadata from insecure
+image registries, such as those signed with a self-signed certificate or using
+plain HTTP instead of HTTPS.
+
+To configure this, add the `*openshift.io/image.insecureRepository*` annotation
+and set it to *true*. This setting bypasses certificate validation when
+connecting to the registry:
+
+====
+[source,yaml]
+----
+kind: ImageStream
+apiVersion: v1
+metadata:
+  name: ruby
+  annotations:
+    openshift.io/image.insecureRepository: "true" <1>
+  spec:
+    dockerImageRepository: my.repo.com:5000/myimage
+----
+<1> Set the `*openshift.io/image.insecureRepository*` annotation to *true*
+====
+
+ifdef::openshift-enterprise,openshift-origin[]
+[IMPORTANT]
+====
+The above definition only affects importing tag and image metadata. For this
+image to be used in the cluster (e.g., to be able to do a `docker pull`), each
+node must have Docker configured with the `--insecure-registry` flag. See
+link:../install_config/install/prerequisites.html#host-preparation[Host
+Preparation] for information.
+====
+endif::[]
+
+Additionally there is an option to specify single tag use insecure repository.
+To do that set `.importPolicy.insecure` in tag's definition to *true*:
+
+====
+[source,yaml]
+----
+kind: ImageStream
+apiVersion: v1
+metadata:
+  name: ruby
+  tags:
+  - from:
+      kind: DockerImage
+      name: my.repo.com:5000/myimage
+    name: mytag
+    importPolicy:
+      insecure: "true" <1>
+----
+<1> Set tag ruby:mytag to use insecure connection to that registry.
+====
+
+[[private-registries]]
+=== Importing Images from Private Registries
+
+An image stream can be configured to import tag and image metadata from private
+image registries, requiring authentication.
+
+To configure this, you need to create a link:../dev_guide/secrets.html[secret]
+which is used to store your credentials.
+
+Create the secret first, before importing image from the private repository:
+
+====
+----
+$ oc secrets new-dockercfg SECRET \
+    --docker-server=DOCKER_REGISTRY_SERVER \
+    --docker-username=DOCKER_USER \
+    --docker-password=DOCKER_PASSWORD \
+    --docker-email=DOCKER_EMAIL
+----
+====
+
+For other options consult with `oc secrets new --help`.
+
+Once the secret is configured proceed with creating new image stream or using
+`oc import-image` command. During the import process {product-title} will pick
+up the secrets and provide to the remote party.
```

Comment:
Maybe it's worth stating the matching is done based on server address, if somebody wonders about that.